### PR TITLE
[CONLT-530] Add slack payload builder for channel managers

### PR DESCRIPTION
**Pull request by kevin-luo-le**
https://aussiecommerce.atlassian.net/browse/CONLT-530

Builds the payload to send to slack for reservation failures occurring on our channel managers

### CODE REVIEW

File: src/slack-notification/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    fields: fields,
+  }
+
+  const payload = {
+    blocks: [
+      {
+        type: 'header',
+        text: {
+          type: 'plain_text',
+          text: 'Siteminder Alert!',
```
Comment (inkiebeard) [inline src/slack-notification/index.ts:37]: Can we make this more configurable?

File: src/slack-notification/index.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { startCase } from 'lodash'
+import { ISlackHookDetails, ISlackHookPayload, ISlackHookPayloadFieldData, ISlackHookPayloadFieldSection } from '../../typings/connections'
+import notifications from '@luxuryescapes/lib-notifications'
+
+const sendSlackNotificationError = async(errorMessage: string, errorDetails: ISlackHookDetails, slackHook: string): Promise<void> => {
+  const text = `${errorDetails.channelManager} failure on request: ${errorMessage}`
+  const payload = buildNotificationPayload(text, errorDetails)
+  notifications.initHook(slackHook, errorDetails.channelManager)
+  await notifications.notify.send.blocks(payload.blocks) }
```
Comment (inkiebeard) [inline src/slack-notification/index.ts:9]: can we reformat this so it's easier to read

File: src/slack-notification/index.ts (hunk 1 above)
Comment (kevin-luo-le) [reply src/slack-notification/index.ts:37]: 🤦 oops yup this was meant to be configurable

**Review by inkiebeard — APPROVED** [review]
🚀

**Pull request merged**